### PR TITLE
Create Migrate_Agent_Valid_Connection page

### PR DESCRIPTION
**Pull request by joaobarraca**
Migrating a Single Agent and Maintaining a Valid Cloud Connection

### CODE REVIEW

**Comment by joaobarraca** [comment]
this page should be put under dataminer.services/Advanced configuration

**Comment by joaobarraca** [comment]
> I'm a little unsure about the current title and location. I think we have two options with this:
> 
> * If we keep the current title & location, we'll need to add the basic migration procedure to the page as well, because otherwise people may get the impression that the current procedure is all they need to do to migrate a DMA.
> * Alternatively, we could move this page below [Restoring a DataMiner Agent](https://docs.dataminer.services/user-guide/Advanced_Functionality/DataMiner_Agents/Restoring_a_DataMiner_Agent.html), and then the title should be e.g. "Maintaining the cloud connection when restoring an Agent on a new server". In that case we'll also need to add a link on the page [Restoring a DMA using the DataMiner Taskbar Utility](https://docs.dataminer.services/user-guide/Advanced_Functionality/DataMiner_Agents/Restoring_a_DMA/Restoring_a_DMA_using_the_DataMiner_Taskbar_Utility.html), to make sure people don't overlook this info.
> 
> Personally, I'm leaning towards this last option. What do you think?

yes, also prefer the last option

File: user-guide/Advanced_Functionality/DataMiner_Agents/Migrating_a_DMA_to_a_new_server.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+1. Copy *CcaPersistedData* from the old server to the new server.
+
+   The DataMiner Agent needs this data in order to maintain its connection to the cloud.
+
+1. Shut down the old server.
+
+    After copying the `CcaPersistedData`, shut down the old server. This is crucial to prevent any conflicts or issues with the cloud connection.
+
+1. Renew the session on the new server.
```
[the remark this answers is not in the document]
Comment (joaobarraca) [reply user-guide/Advanced_Functionality/DataMiner_Agents/Migrating_a_DMA_to_a_new_server.md:20]: info can be found here:
https://docs.dataminer.services/user-guide/Reference/DataMiner_Tools/SLNetClientTest_tool/SLNetClientTest_tool_advanced_procedures/SLNetClientTest_debugging_cloud_connection.html

File: user-guide/Advanced_Functionality/DataMiner_Agents/Migrating_a_DMA_to_a_new_server.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+When you migrate a DataMiner Agent to a new server, you need to follow the steps below to make sure its connection to dataminer.services remains valid.
+
+> [!NOTE]
+> This procedure only applies for DataMiner Agents that are connected to dataminer.services. If your DMA is not connected yet, we highly recommend that you [connect to dataminer.services](xref:Connecting_your_DataMiner_System_to_the_cloud) after the migration in order to gain access to additional DataMiner features.
+
+> [!TIP]
+> See also: [Changing the IP of a DMA](xref:Changing_the_IP_of_a_DMA)
+
+1. Copy *CcaPersistedData* from the old server to the new server.
```
[the remark this answers is not in the document]
Comment (joaobarraca) [reply user-guide/Advanced_Functionality/DataMiner_Agents/Migrating_a_DMA_to_a_new_server.md:12]: C:\ProgramData\Skyline Communications\DataMiner CloudGateway\Data\CcaGatewayPersisted

**Comment by MariekeGO** [comment]
@joaobarraca, I've adjusted the PR as discussed. Could you check if everything is still OK for you?

**Review by joaobarraca — COMMENTED** [review]
looks good to me

**Pull request merged**